UEBPrep/app/controllers/cards_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CardsController < ApplicationController
+  def new
+    @user = current_user
```

Comment:
Should just be able to check if not nil.... by @user.nil? will return true if nil, false otherwise. Also this is a point where we can make a reasonable assumption that you can't get to this page without being logged in, which we can use devise to ensure happens-by placing a callback in this controller that checks that the user is logged in before taking any action.